vsphere/internal/helper/virtualmachine/virtual_machine_helper.go

```diff
@@ -367,6 +367,40 @@ func skipIPAddrForWaiter(ip net.IP, ignoredGuestIPs []interface{}) bool {
 	return false
 }
 
+func blockUntilReadyForMethod(method string, vm *object.VirtualMachine, waitFor time.Duration) error {
+	log.Printf("[DEBUG] blockUntilReadyForMethod: Going to block until %q is no longer in the Disabled Methods list for vm %s", method, vm.Reference().Value)
```

Comment:
Is there a way to configure logging by log level so this isn't always printed?

Reply:
The config library will take care of that.